spec/workflow_service_spec.rb

```diff
@@ -586,4 +587,31 @@
       expect(Dor::WorkflowService.get_lane_ids('dor', 'accessionWF', 'shelve')).to eq(%w(lane1 lane2))
     end
   end
+
+  describe '.workflow_resource_method' do
+    let(:stubs) do
+      Faraday::Adapter::Test::Stubs.new do |stub|
+        stub.get('x?complete=a&complete=b') do |env|
+          [200, {}, 'ab']
+        end
+      end
+    end
```

Comment:
This seems like a `before :each` block rather than a `let` because the `let` is lazy evaluated and I don't see any calls to `stubs` in the spec anywhere so it's not obvious that this stuff actually executes.